### PR TITLE
CTDA-1358/new-917-xsd

### PR DESCRIPTION
**Pull request by GeorgeColdham**
# CTDA-1358/new-917-xsd

[Ticket](https://jira.tools.tax.service.gov.uk/browse/CTDA-1358)

### CODE REVIEW

File: source/documentation/xsd-reference.html.md.erb [hunk 1]
```diff
@@ -20,41 +20,43 @@ Use these to validate your XML. Please note, there are some known omissions. The
...
+|**[Arrival notification message CC007A (IE007)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc007a.xsd)** |Message type sent by the trader to the office of departure containing an arrival notification|
+|**[Declaration data cancellation request CC014A (IE014)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc014a.xsd)**|Message type sent by the trader to the office of departure containing a cancellation request|
+|**[Declaration data message CC015B (IE015)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc015b.xsd)**|Message type sent by the trader to the office of departure to indicate that they would like to start a movement|
+|**[Unloading remarks message CC044A (IE044)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc044a.xsd)** |Message type sent by the trader to the office at destination to let them know that the goods have been unloaded (IE044)|
```
Comment (DavidGregory084) [inline source/documentation/xsd-reference.html.md.erb:34]: I know it has always been like this but should we really be pointing people to `master`?

**Review by DavidGregory084 — APPROVED** [review]
👍

File: source/documentation/xsd-reference.html.md.erb [hunk 2]
```diff
@@ -20,41 +20,43 @@ Use these to validate your XML. Please note, there are some known omissions. The
 
 You can ignore this field.
 
-### [Download zip file](ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
+### [Download zip file](https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide/documentation/ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
```
Comment (crystalmark) [inline source/documentation/xsd-reference.html.md.erb:23]: Can we make the url relative?  If it contains the hostname, it probably will not work in QA.

File: source/documentation/xsd-reference.html.md.erb (hunk 1 above)
Comment (GeorgeColdham) [reply source/documentation/xsd-reference.html.md.erb:34]: Unless they are hosted elsewhere, what options do we have?

File: source/documentation/xsd-reference.html.md.erb (hunk 2 above)
Comment (GeorgeColdham) [reply source/documentation/xsd-reference.html.md.erb:23]: So `./documentation/ctc-traders-api-xsds.zip`?

File: source/documentation/xsd-reference.html.md.erb (hunk 1 above)
Comment (DavidGregory084) [reply source/documentation/xsd-reference.html.md.erb:34]: We could link to a recent tag, e.g. https://github.com/hmrc/common-transit-convention-traders/blob/v0.240.0/conf/xsd/cc014a.xsd

**Pull request merged**